README.md

```diff
@@ -70,3 +70,32 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+
+## Using Vagrant
+You can create a new Hypervisor installed with your app and in running mode.
+* Download and install VirtualBox on your host https://www.virtualbox.org/wiki/Downloads
+* Download and install Vagrant: https://www.vagrantup.com/downloads 
+* Open powershell 
+* Go to the root of your TODO app
+* Run the following on your command line to create the hypervisor with the TODO app installed and running:
```

Comment:
Just a correction with the terminology - you are creating a VM, not a hypervisor. 

A hypervisor is the software that handles the virtualisation/runs the VMs.

Reply:
Sorry yes - agreed.